src/components/Pricing/PlanComparison/index.tsx

```diff
@@ -55,7 +55,7 @@ const sections = [
                 name: 'Scales to...',
                 tiers: {
                     'PostHog Cloud': 'Millions of users/mo',
-                    'Open source': '~1m users/mo',
+                    'Open source': '~100k users/mo (no hard limit, but we recommend support beyond this)',
```

Comment:
What does "support" mean here? Reads a little vague to me. Does the customer need support to scale it? Do they need to get support from us?